scripts/automated_ingestion/automated_ingestion.py

```diff
@@ -100,15 +100,17 @@ def main():
         aws_secret_access_key=config['secrets']['aws_secret_access_key'],
     )
 
-    tarballs = find_tarballs(s3, config['aws']['staging_bucket'])
-    if args.list_only:
-        for num, tarball in enumerate(tarballs):
-            print(f'{num}: {tarball}')
-        sys.exit(0)
-
-    for tarball in tarballs:
-        tar = EessiTarball(tarball, config, gh, s3)
-        tar.run_handler()
+    buckets = [x.strip() for x in config['aws']['staging_buckets'].split(',')]
+    for bucket in buckets:
+        tarballs = find_tarballs(s3, bucket)
+        if args.list_only:
+            for num, tarball in enumerate(tarballs):
+                print(f'{num}: {tarball}')
+            sys.exit(0)
```

Comment:
This was already a bit ugly, but doesn't work as intended anymore at all now, as it will exit before the second bucket is being handled. Easy fix would be to change it in `continue`, but maybe it should be done a bit cleaner with a proper `if`/`else` or something

Reply:
fixed in aed065a